priv/templates/extended_bin

```diff
@@ -115,7 +140,34 @@ relx_usage() {
                 relx_run_extension "$EXTENSION_SCRIPT" help
             else
                 EXTENSIONS=$(echo $EXTENSIONS | sed -e 's/|undefined//g')
-                echo "Usage: $REL_NAME {start|start_boot <file>|foreground|stop|restart|reboot|pid|ping|console|console_clean|console_boot <file>|attach|remote_console|upgrade|downgrade|install|uninstall|versions|escript|rpc|rpcterms|eval|status|$EXTENSIONS}"
+                echo "Usage: ${REL_NAME} [COMMAND] [ARGS]
+
+Commands:
+
+  foreground              Start release with output to stdout
+  remote_console          Connect remote shell to running node
+  rpc [Mod [Fun [Args]]]] Run apply(Mod, Fun Args) on running node
+  eval [Exprs]            Run expressions on running node
+  stop                    Stop the running node
+  restart                 Restart the applications but not the VM
+  reboot                  Reboot the entire VM
+  pid                     Print the PID of the OS process
+  ping                    Print pong if the node is alive
+  console                 Start the release with an interactive shell
+  console_clean           Start an interactive shell without the release's applications
+  console_boot [File]     Start an interactive shell for boot script [File]
+  daemon                  Start release in the background with run_erl
+  daemon_boot [File]      Start boot script [File] in the background with run_erl
+  daemon_attach           Connect to node started as daemon with to_erl
```

Comment:
Is `daemon` mode the one using named pipes? If so it might be useful to mention something like `... with run_erl (named pipes)`. I might otherwise have expected it to just be using `-detached` as a flag there, unless there'd be a `background` one (matching with `foreground` and working with `remote_console`).

Reply:
Yea, it is. I'll add `(named pipes)` to the description.

We could name it `run_erl` instead of `daemon`, to be more explicit. That would also cause people to actually look up `run_erl` before using it. `daemon` like `start` sounds too generic to cause someone to look into what exactly it will be doing.